packages/apps/demo/src/pages/Browse.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2023 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Base, Column } from '@lightningjs/ui-components';
+import BackgroundSwitcher from '../components/BackgroundSwitcher';
+import { titleRowFormatter } from '../formatters';
+
+export default class Browse extends Base {
+  static get __componentName() {
+    return 'Browse';
+  }
+
+  static _template() {
+    return {
+      Hero: {
+        type: BackgroundSwitcher
+      },
+      Results: {
+        subTheme: 'rowSubTheme',
+        type: Column,
+        boundsMargin: [ 200, 200, 0, 0 ],
```

Comment:
maybe add a code comment explaining expanding the view port for data loading reasons?